hail/python/hail/plot/plots.py

```diff
@@ -245,11 +245,11 @@ def qq(pvals, collect_all=False, n_divisions=500):
                 obs = [-log(p, 10) for p in spvals]
             else:
                 if isinstance(source, Table):
-                    ht = source.select(pval=pvals)
+                    ht = source.key_by(pval=pvals)
```

Comment:
is the select here unnecessary because of pruning?

Reply:
oh, yeah. meant to .select() after

Reply:
we should really prune that automatically, but I don't think we're doing that now